packages/terminal/src/browser/base/terminal-widget.ts

```diff
@@ -16,6 +16,8 @@
 
 import { Event } from '@theia/core';
 import { BaseWidget } from '@theia/core/lib/browser';
+import { TerminalProcessInfo } from '../../common/base-terminal-protocol';
+import { CommandLineOptions } from '@theia/process/libcommon/run-in-terminal';
```

Comment:
`lib/common`

Reply:
Somehow this is the works of TypeScript's auto-import, idk why it keeps doing that???